color/src/missing.rs

```diff
@@ -3,45 +3,49 @@
 
 //! A simple bitset.
 
-/// A simple bitset, for representing missing components.
+/// A simple bitset for representing missing components.
 #[derive(Default, Clone, Copy, PartialEq, Eq, Debug)]
-pub struct Bitset(u8);
+pub struct Missing(u8);
```

Comment:
Would it be clearer if this represented present (non-missing) components? 0 to represent "present" and 1 to represent "missing" seems somehow inverted. Presumably this is equivalent on a technical level?

Reply:
Perhaps "missing" semantics make sense because the common case is "no missing components"? If so then perhaps the code could stay the same, but the method could be documented in terms of "which components are present (non-missing)" rather than (or in addition to) "which components are in the set".

e.g. for `is_empty`:

```
Returns `true` if the set contains no indices (and therefore all color components are present).
```

Reply:
Would also appreciate an/some example(s) of when components can be missing.

Reply:
I'll add some docs about missing components.

Inverting the boolean sense is equivalent at a technical level. I think the strongest case for this way ("missing") is that the default value of 0 makes sense.

Incidentally, I considered and rejected some other representations for this. In particular, [color.js](https://colorjs.io/docs/interpolation) uses `NaN`. That works in pure Rust, but I have at least two problems with it: it complicates GPU interop, because NaN is almost always a bad thing), and it makes the test for "any missing components" more expensive, because you have to do a finiteness check on all the components, as opposed to comparison with 0 here. The current approach also has no memory cost, `DynamicColor` is 20 bytes either way.